packages/react/src/components/Checkbox/Checkbox.stories.tsx

```diff
@@ -149,71 +149,83 @@ export const Disabled = {
   render: Group,
 };
 
-export const InTable: StoryFn<UseCheckboxGroupProps> = (args) => {
+const tableData = [
+  {
+    id: 1,
+    navn: 'Lise Nordmann',
+    epost: 'lise@nordmann.no',
+    telefon: '22345678',
+  },
+  {
+    id: 2,
+    navn: 'Kari Nordmann',
+    epost: 'kari@nordmann.no',
+    telefon: '87654321',
+  },
+  {
+    id: 3,
+    navn: 'Ola Nordmann',
+    epost: 'ola@nordmann.no',
+    telefon: '32345678',
+  },
+  {
+    id: 4,
+    navn: 'Per Nordmann',
+    epost: 'per@nordmann.no',
+    telefon: '12345678',
+  },
+];
```

Comment:
```suggestion
  {
    id: 1,
    navn: 'Lise Nordmann',
    epost: 'lise@nordmann.no',
    telefon: '68051156',
  },
  {
    id: 2,
    navn: 'Kari Nordmann',
    epost: 'kari@nordmann.no',
    telefon: '68059679',
  },
  {
    id: 3,
    navn: 'Ola Nordmann',
    epost: 'ola@nordmann.no',
    telefon: '68055731',
  },
  {
    id: 4,
    navn: 'Per Nordmann',
    epost: 'per@nordmann.no',
    telefon: '68059631',
  },
];
```
Lat oss bruke telefonummer som er faktisk fiktive :)
Sjå: https://nkom.no/telefoni-og-telefonnummer/telefonnummer-og-den-norske-nummerplan/alle-nummerserier-for-norske-telefonnumre#:~:text=Nummer%20for%20TV%2D%20og%20filmproduksjon